script/ContractAddresses.sol

```diff
@@ -112,10 +112,10 @@ contract ContractAddresses {
                 STETH_STRATEGY_ADDRESS: 0x7D704507b76571a51d9caE8AdDAbBFd0ba0e63d3 // Placeholder address, replaced with address(1) for holesky
             }),
             yn: YieldNestAddresses({
-                YNETH_ADDRESS: 0xd9029669BC74878BCB5BE58c259ed0A277C5c16E,
-                STAKING_NODES_MANAGER_ADDRESS: 0xc2387EBb4Ea66627E3543a771e260Bd84218d6a1,
-                REWARDS_DISTRIBUTOR_ADDRESS: 0x82915efF62af9FCC0d0735b8681959e069E3f2D8,
-                EXECUTION_LAYER_RECEIVER_ADDRESS: 0xA5E9E1ceb4cC1854d0e186a9B3E67158b84AD072,
+                YNETH_ADDRESS: 0xe8A0fA11735b9C91F5F89340A2E2720e9c9d19fb, //0xd9029669BC74878BCB5BE58c259ed0A277C5c16E,
```

Comment:
we should not apply this change to this file, since the pre-prod deployment does not turn into a canonical deployment